update_compilers/install_compilers.sh

```diff
@@ -536,3 +536,54 @@ fi
 cp ${SCRIPT_DIR}/custom/fpc.cfg ${OPT}/fpc/fpc.cfg
 
 #########################
+# NASM
+
+get_nasm() {
+    local VER=$1
+    local DIR=nasm-$VER
+
+    if [[ ! -d ${OPT}/${DIR} ]]; then
+        pushd /tmp
+        fetch http://www.nasm.us/pub/nasm/releasebuilds/${VER}/nasm-${VER}.tar.xz | tar Jxf -
+        cd ${DIR}
+        sh configure
+        make
+        mkdir ${OPT}/${DIR}
+        cp nasm ${OPT}/${DIR}
+        popd
+    fi
+}
+
+for version in \
+    2.12.02 \
+    2.13.02 \
+; do
+    get_nasm $version
+done
+
+#########################
+# GNU AS
+
+get_gnuas() {
+    local VER=$1
+    local DIR=gnuas-$VER
+
+    if [[ ! -d ${OPT}/${DIR} ]]; then
+        pushd /tmp
+        fetch http://ftp.gnu.org/gnu/binutils/binutils-${VER}.tar.xz | tar Jxf -
```

Comment:
Gnu's ftp is available as https

Reply:
Now i'm thinking about this, this might just be installed along with gcc... can you see if you can find 'as' within the /opt/compiler-explorer/gcc... directories?

Reply:
gcc 7.2.0 does have `as` under `bin`, yes!

Reply:
Ok then i'll change it to use that like I did with the Clang version